spec/features/see_reviews_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+feature "see reviews", %{
+  As a user,
+  I want to view the reviews for a beer
+  So that my mind can be influenced
+} do
+  # Acceptance Criteria
+  # * I must be on the details page for a specific beer
+  # * I must only see the reviews for the beer I’m viewing
+  # * I must see the reviews listed in order, most recent first
+
+  scenario "sucessfully see reviews" do
+    reviews = Factorygirl.create_list(:review, 2)
+    unrelated_review = Factorygirl.create(:review)
+
+    visit root_path
+    click_link reviews.first.beer
+
+    expect(page).to have_selector(
```

Comment:
Using have_selector in order to check for the correct order of the reviews (i.e. most recent first). This is the best approach that I found on google/stack overflow to accomplish this. Is there a better way??